### PR TITLE
executor: notify TiDB update privilege after alter/drop user

### PR DESCRIPTION
**Pull request by tiancaiamao**
Alter User and Drop User forget to do it.

Fix https://github.com/pingcap/tidb/issues/5216

### CODE REVIEW

**Comment by breeswish** [comment]
LGTM

File: executor/simple.go [hunk 1]
```diff
@@ -219,6 +219,7 @@ func (e *SimpleExec) executeAlterUser(s *ast.AlterUserStmt) error {
 		errMsg := "Operation ALTER USER failed for " + strings.Join(failedUsers, ",")
 		return terror.ClassExecutor.New(CodeCannotUser, errMsg)
 	}
+	domain.GetDomain(e.ctx).NotifyUpdatePrivilege(e.ctx)
```
Comment (shenli) [inline executor/simple.go]: If we meet error here, add a log.

Comment (tiancaiamao) [reply executor/simple.go]: NotifyUpdatePrivilege already log errors.

**Comment by shenli** [comment]
LGTM

**Pull request merged**